src/qforte/system/molecular_info.py

```diff
@@ -84,6 +84,17 @@ class Molecule(System):
         It contains the RHF orbital energies (in hartree) of the system, as computed by Psi4.
         In the case of H2/STO-3G, r = 3.0 angs, for example: hf_orbital_energies = [-0.1805392218304829, 0.018071329565966215]
 
+    ###### Attributes pertaining to frozen-orbital approximations ######
```

Comment:
I'm not convinced that grouping attributes like this is worth the price of being able to alphabetize keywords. We can save that for later. QForte will need its documentation greatly improved...